README.md

```diff
@@ -1,21 +1,27 @@
-# Alexander performance tests
-## a.k.a Alexander the great
-Here we can found the framework and definition for every performance test suite defined
+# Alexander Performance Tests
```

Comment:
Why alexander? I would change the name of this....

Reply:
Updated to `# Mozart Performance Tests`. Keep in mind this was created to perf-test alexander and then repurposed as a "Let's test all of mozart" tool. Needs lots of love and a VISION of what we want from this.